src/code/z_player_lib.c

```diff
@@ -1201,7 +1372,7 @@ s32 func_801240DC(Player* player) {
     return Player_IsHoldingHookshot(player) && (player->heldActor == NULL);
 }
 
-s32 func_80124110(Player* player, s32 actionParam) {
+s32 func_80124110(Player* player, PlayerActionParam actionParam) {
     s32 temp_v0 = actionParam - PLAYER_AP_FISHING_POLE;
```

Comment:
I feel like this is not the correct calculation, but there's still a lot to do here so I'm not going to worry for now.